eitprocessing/binreader/sequence.py

```diff
@@ -11,141 +11,206 @@
 import itertools
 from dataclasses import dataclass
 from dataclasses import field
+from enum import auto
 from pathlib import Path
 from typing import Dict
 from typing import List
 from typing import Tuple
 import numpy as np
+from strenum import LowercaseStrEnum
 from .event import Event
 from .frameset import Frameset
 from .phases import MaxValue
 from .phases import MinValue
 from .phases import PhaseIndicator
+from .phases import QRSMark
 from .reader import Reader
 from .timing_error import TimingError
 
 
-@dataclass
+class Vendor(LowercaseStrEnum):
+    DRAEGER = auto()
+    TIMPEL = auto()
+    SENTEC = auto()
+    DRAGER = DRAEGER
+    DRÄGER = DRAEGER
+
+
+@dataclass(eq=False)
 class Sequence:
-    path: Path
+    path: Path | str = None
     time: np.ndarray = None
     n_frames: int = None
-    framerate: int = 20
+    framerate: int = None
     framesets: Dict[str, Frameset] = field(default_factory=dict)
     events: List[Event] = field(default_factory=list, repr=False)
     timing_errors: List[TimingError] = field(default_factory=list, repr=False)
     phases: List[PhaseIndicator] = field(default_factory=list, repr=False)
+    vendor: Vendor = None
```

Comment:
```suggestion
    vendor: Vendor | str = None
```

Also, make sure that old tests still work or get updated when you change functions/classes.
The "test_my_module" test file is not giving this variable.

Reply:
I fixed this in f76e2513.